src/slices/tasksSlice.js

```diff
@@ -0,0 +1,76 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
+import { saveTasks, loadTasks } from '../repositories/taskRepository';
+
+const initialState = {
+  tasks: loadTasks().tasks || [],
+  tasksOrder: loadTasks().tasksOrder || [],
```

Comment:
Do you really need separate storage for tasks order? Could you please re-order tasks and keep ordering in the state?

Reply:
removed taskOrder